source/class/qx/ui/table/Table.js

```diff
@@ -2193,7 +2193,6 @@ qx.Class.define("qx.ui.table.Table", {
       );
     }
 
-    this._cleanUpMetaColumns(0);
```

Comment:
It's not clear to me how this fixes the described problem. `__scrollerParent` was already being disposed at line 2200, so even if `_cleanUpMetaColumns` caused a delayed action, wouldn't those event listeners be cleaned up when `__scrollerParent` is disposed anyway?

Reply:
@derrell - I think when `_cleanUpMetaColumns` is called, and `destroy` is called for each Scroller, any event listeners are immediately removed from the Scroller itself, but not from it's child components. The `_paneClipper` child has this particular pointermove listener and so that would not get removed until disposal. Also, I think `destroy` removes the Scroller from it's parent `__scrollerParent` so when it is actually disposed, the Scrollers are still delayed. In our app, it is very easy to replicate the error, there is an "are you sure" dialog appears in the foreground to delete an item, when you click yes there is a backend call and then a call to destroy the table. There is a small delay in the dialog disappearing and in that time, you can wiggle the mouse over the table and 2 out of 3 times you could generate the error. With the change, I can't generate the error at all.

Reply:
My concern is that `_cleanUpMetaColumns` was freeing the memory for each of `__scrollerParent`'s children, which is no longer being done. (`__scrollerParent` is a `qx.ui.container.Composite` which itself does not have a `destruct` method.) Now it may be unnecessary to explicitly clean that up, assuming that modern memory management will handle the cleanup. Is my concern unfounded? I just want to ensure we're not adding a memory leak here... 

@johnspackman You implemented the changes a year or so ago that negated the need for many `destruct` calls. What is the criteria, these days, for when it's needed and when it's not?

Reply:
Oh! However, https://github.com/qooxdoo/qooxdoo/blob/b4e50a92b39b71cb5b49619694fa5c6121c4d1cd/source/class/qx/ui/core/Widget.js#L3669 disposes the children. I think we're good.